awx/ui_next/src/setupTests.js

```diff
@@ -24,3 +24,10 @@ global.console = {
 // and so this mock ensures that we don't encounter a reference error
 // when running the tests
 global.__webpack_nonce__ = null;
+
+// This is to make websocket-using components behave as if they're connecting via
+// wss, which is the default behavior being tested in most websocket tests
+Object.defineProperty(window, 'location', {
+  value: new URL('https://localhost'),
+});
+window.location.replace = jest.fn();
```

Comment:
This may be a bit heavy-handed, but the mock to work fine for the tests we have so far. :man_shrugging: